.golangci.yml

```diff
@@ -66,4 +73,4 @@ issues:
         - funlen           # tests may be long
         - gomnd            # tests are full of magic numbers
         - testpackage      # senseless
-        - unused           # very annoying false positive: https://github.com/golangci/golangci-lint/issues/791
\ No newline at end of file
+        - unused           # very annoying false positive: https://github.com/golangci/golangci-lint/issues/791
```

Comment:
btw looks like the false positive issue in the comment (https://github.com/golangci/golangci-lint/issues/791) is fixed now.

Reply:
Removed that line, however `unused` is disabled for 1.18 automatically